Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -802,9 +793,7 @@ void WebProcessProxy::updateBackForwardItem(const BackForwardListItemState& item
 void WebProcessProxy::getNetworkProcessConnection(Messages::WebProcessProxy::GetNetworkProcessConnection::DelayedReply&& reply)
 {
     auto* dataStore = websiteDataStore();
-    if (!dataStore)
-        return reply({ });
-
+    RELEASE_ASSERT(dataStore);
```

Comment:
This looks very dangerous. If this happened, we'd be converting a WebContent process crash into a UIProcess crash, which is not a good trade off IMO. I'd recommend a debug ASSERT() instead and a RELEASE_LOG_FAULT() so we know it's happening.